src/views/progressionbar.js

```diff
@@ -10,9 +11,9 @@ export const progressionBar = (currentIndex) => {
 
   userInterface.appendChild(progressionDiv);
   progressionDiv.appendChild(progress);
-  let num = 21.5;
+  let num = 43;
```

Comment:
please avoid using magic numbers next time

Reply:
noted...will remmeber this...
thanks for the comment...